lib/state_machines/integrations/active_record.rb

```diff
@@ -24,14 +24,22 @@ module Integrations #:nodoc:
     # 
     # == Actions
     # 
-    # By default, the action that will be invoked when a state is transitioned
-    # is the +save+ action.  This will cause the record to save the changes
```

Comment:
Perhaps this is referring to `#ignite!` vs `#ignite`.  I'm not sure the old documentation is clear, but I don't think that the new doc is necessarily good guidance either.  I'm wondering if there are references elsewhere to clear this up?  I agree the docs need to change, just not sure to what.  I use the no-bang and bang methods as needed knowing that save will or won't be triggered.